### ISSUE TITLE
Choose hypothesis for first intervention

### ISSUE THREAD

**Issue by secretrobotron** [MozillaFoundation/digital-skills-observatory#97]
We've done a lot of work to refine the list of [research questions](https://github.com/MozillaFoundation/digital-skills-observatory/blob/master/questions.md) and build some hypotheses based on them to help us develop the first intervention. A fuller (read: messier) list is in the [DSO Master Doc](https://docs.google.com/spreadsheets/d/1-UPaBLNLAbqYZ-u3QQIxdPWT7MgJ6cSXBphCTWbVY4g/edit?pref=2&pli=1#gid=2146668059), but I thought it would be useful to lift the details out and put them here for clearer discussion.
### Some notes
- We've chosen the list of hypotheses below based on lots of conversation and opinion and reading and talking. They are good candidates for our **first** intervention, but [the list of hypotheses](https://docs.google.com/spreadsheets/d/1-UPaBLNLAbqYZ-u3QQIxdPWT7MgJ6cSXBphCTWbVY4g/edit?pref=2&pli=1#gid=2146668059) is still open for additions, comments, etc., which we can use to decide on the next intervention.
- There is a bunch of potential for these hypotheses to cascade. e.g. If we start by teaching people about smartphone ecosystems, we can teach them about app- or DFS-specific ecosystems afterward. 
- These are "hypotheses" in a loose sense: they are here to anchor our thinking in the design process of the intervention, but not as elements we will strive to prove or disprove with scientific rigor. Our objective is to observe and watch for insights that will contribute to the overall project narrative, or future prospective research. (e.g. Unless it's incredibly obvious, we won't state something like, "learning about X influenced Y% of people," but we can say, "many people who reflected on the interventions mentioned that Z was really helpful, and we should pursue that further.")
### HYPOTHESES
1. If we teach people about smartphones, their ecosystems, and the economics around smartphones, then their financial decisions are affected. _(Relates to research questions 7 & 8)_
2. If we teach people about apps and app ecosystems, then they are more likely to find and install apps on their own. _(Relates to research question 9)_
   1. If we teach people about apps and app ecosystems, then they are more likely to find and install DFS apps on their own.
   2. If we teach people about apps and app ecosystems, then they are no more likely to install DFS apps on their own.  but more likely to install entertainment apps e.g. The Bible, games, social media, etc.
   3. If we teach people about apps and app ecosystems AND DFS, then they are more likely to find and install DFS apps on their own.
3. If people receive digital skills training from an "authority" (as opposed to friends, or someone related to finance), then they will feel more comfortable using smartphones _(Relates to research questions 13, 11 & 12)_
   1. If people receive digital skills training from an "authority", then they will feel more comfortable using DFS.
   2. If people receive digital skills training from an "authority", then they will feel more confident using DFS.
4. If we teach people digital skills, then they will be more confident and feel more agency in their financial lives.
### What do you think?

If you've got an opinion about which hypothesis we should pursue for our first intervention, let's talk about it here!

**Comment by secretrobotron** [comment]
**RECOMMENDATION**: @LauraReynal and I have been asking people to reflect on the hypotheses a bunch, and we really like the possibilities that the first hypothesis affords us as we think about the design of the first intervention. It will let us be broad at first, and get more specific in the future.

**Comment by chanditome** [comment]
Hypothesis 1 is looks broad and I do not understand Hypothesis 4. 
Hypothesis 3 puts us (the "authority") in the picture, I assume. I am reserved about that at the moment. 
Hypothesis 2 is in my opinion our best shot for as we begin because I think it's more testable than the rest.

**Comment by secretrobotron** [comment]
@chanditome yeah, I think you're right about H4. We need do build more of an understanding and opinion about our respondents before we can attack that.

For H3, the idea wasn't that we'd be the authority. It could be anyone, and could take any form that people associate with having lots of formal knowledge and the answers to your questions.

For H1/2: the first is intentionally broad, and the second definitely follows it. Our idea was to start with H1 and move to H2 next. From other research and some preliminary testing, it looks like we might need to start in a place that illustrates broader ideas about smartphones/infrastructure. The results might not be hugely positive (but I would love to be pleasantly surprised!), but it would give us a solid platform on top of which we can build much stronger interventions with more specific goals.

**Comment by lauradereynal** [comment]
@chanditome thanks for the feedback ! 

H1 is really about understanding the smartphone ecosystem in the sense: what it is, how it works, how it connects to the web, who are the different actors... there is a strong economic dimension in this one where people would learn the economics around the smartphones. 
Let me rephrase H1 to reflect that a bit more.

**Comment by lauradereynal** [comment]
Also, I'd like to hear feebdack and input from @denomain @ChristabelAchieng @WinnieMakokha @zuma1 @xelawafs @FauziaIsmail @festolangat @Amuti ... and many others :smile: Feel free to tag a friend?

**Comment by AI-DennisNdegwa** [comment]
I think H1 would give insight into more things that are not currently known as compared to say H2 and H3. I think H4 makes sense too. In the sense that most people are afraid of venturing into new DFSs like paypal and pesapal simply due to the fear of loosing their money.

**Comment by secretrobotron** [comment]
:+1: thanks @denomain. Perhaps H4 would be useful to weave into the second intervention.

With a H1 as a basis, using H4, we could develop ways to let people to understand and use those DFS's more easily. (To me, it sounds like there might be some interesting software prototypes to develop in there.)

**Comment by simeonoriko** [comment]
Last year in my digital literacy research at Equity Foundation, I found that Digital Literacy can't be pursued on it's own - it needs to be anchored in a specific context or drive at a specific goal.

In this light, H1 makes sense - the context is the individuals financial situation and that should lend itself to useful outcomes.

I also think H4 makes sense  - for the same reason as H1 but also because it moves the user from knowledge to taking practical steps (e.g. downloading and using apps)

**Comment by lauradereynal** [comment]
Thanks for your input @simeonoriko 
I really like your point about digital literacy. Can you share your research and key findings with this group? it's definitely worth a quick mail on the list, and a note in our Bookclub folder here: https://docs.google.com/document/d/1Da3BLSPhILhJQCjPllvh33BGVmspi6Tr4gTdu4hjb3Y/edit#

**Comment by lauradereynal** [comment]
I've been thinking about H2 and think it should be moved to later in the year, otherwise it will biased all of our results: it's generally too centered on DFS apps and should be rephrased for just "Navigating the appstore". 

Thoughts?

**Comment by Amuti** [comment]
I conquer with what @simeonoriko says in his first phrase that when you are teaching people digital literacy you have to be specific and anchor to a a specific context however with reference to this, according to me -:

H2 seems more appropriate since the specific interaction users have with smartphones is apps so it would be better to center our intervention towards that

H4 would be also appropriate in filling in the users gaps of knowledge as a whole. They might not be accessing DFS solely via smartphones so by teaching them basic digital skills like PC & Internet usage they might be able to understand and use other DFS e.g internet banking services, buying insurance online, paying for goods online etc

I hope I am making sense?

**Comment by AI-DennisNdegwa** [comment]
@secretrobotron I agree. H4 would create a good basis for the second intervention. It kind of hits the DFS angle of the research on the head. I believe we'll be able to understand more in terms of the current situation and the possibility of the study group using more DFS in the future.

**Comment by lauradereynal** [comment]
Thanks for the contribution everyone. After a review with community members + experts + informal advisors + core team, we are going to proceed with Hypothesis 1 for intervention 1. It is broad and shall cover interesting topics. Also, we are learning from the field that this will be useful to teach people.
